bot.py

```
@@ -25,11 +26,122 @@
+
+    async def serialize(self, frame):
+        # Only log TextFrames, not audio frames
+        if not (hasattr(frame, '__class__') and frame.__class__.__name__ == 'OutputAudioRawFrame'):
+            logger.debug(f"Serializing frame type: {type(frame).__name__}")
+        
+        # Special handling for TranscriptionFrame 
+        if isinstance(frame, TranscriptionFrame):
+            try:
+                # Create a field with both formats to ensure client can read it
+                frame.user_id = 'ai'  # Standard snake_case format
+                logger.debug(f"Set TranscriptionFrame user_id to 'ai'")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)